import 'package:futures_then_catcherror_whencomplete/futures_then_catcherror_whencomplete.dart';
import 'package:test/test.dart';

void main() {
  test('calculate', () {
    expect(calculate(), 42);
  });
}
